overlays/nodejs/lambda/lambdaMixins.ts

```diff
@@ -0,0 +1,302 @@
+// Copyright 2016-2018, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as pulumi from "@pulumi/pulumi";
+import * as crypto from "crypto";
+
+import * as arn from "../arn";
+import * as iam from "../iam";
+import * as runtime from "./runtimes";
+import * as utils from "../utils";
+
+import * as lambdaFunction from "./function";
+import * as permission from "./permission";
+
+/**
+ * Context is the shape of the context object passed to a Function callback.
+ */
+export interface Context {
+    callbackWaitsForEmptyEventLoop: boolean;
```

Comment:
Nit, wish we had comments for these properties.

Reply:
Docs added.